pkg/serviced.default

```diff
@@ -163,34 +156,10 @@
 # To disable snapshot removal, set the value to 0.
 # SERVICED_SNAPSHOT_TTL=12
 
-# The interval, in seconds, at which a serviced instance configured as agent
-# modifies its /opt/serviced/var/monitor/<IP-Address> file.
-# Default: 60     (1 minute)
-# SERVICED_MONITOR_DFS_REMOTE_UPDATE_INTERVAL=60
-
-# The interval, in seconds, at which low-level storage stats will be pulled and
-# analyzed from the devicemapper kernel driver table.
-# SERVICED_STORAGE_STATS_UPDATE_INTERVAL=300
-
-# The interval, in seconds, at which a serviced instance configured as master
-# checks the modification times of the /opt/serviced/var/monitor/<IP-Address>
-# files of active serviced instances configured as agents. The value of this
-# variable must be a minimum of twice the value of the
-# SERVICED_MONITOR_DFS_REMOTE_UPDATE_INTERVAL variable.
-# Default: 180    (3 minutes)
-# SERVICED_MONITOR_DFS_MASTER_INTERVAL=180
-
-# Set to 0 in order to disable restarting nfs upon out of sync detection
-# Default: 1    (enable)
-# SERVICED_MONITOR_DFS_MASTER_RESTART=1
-
 # Set to 0 in order to prevent this host from attempting to mount the DFS
 # Default: 1
 # SERVICED_NFS_CLIENT=1
 
-# Overrides the default for the service migration image.
-# SERVICED_SERVICE_MIGRATION_TAG=1.0.2
```

Comment:
SERVICED_SERIVCE_MIGRATION is still used in  facade/service.go. Please do not remove.